browser/templates/browser/investigator_delete_modal.html

```diff
@@ -0,0 +1,24 @@
+<div class="modal fade" id="InvestigatorDeleteModal" tabindex="-1" role="dialog" aria-labelledby="deleteModalLabel">
+  <div class="modal-dialog" role="document">
+    <div class="modal-content">
+      <div class="modal-header">
+        <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+        <h4 class="modal-title" id="deleteModalLabel">Delete Investigation record</h4>
+      </div>
+      <div class="modal-body">
+
+        <h5>
+        Are you sure you want to delete following data for <strong>{{person.name}}</strong> : <br><br>
```

Comment:
why would there be a blank between the question and the colon?

Reply:
Removed the blank between the question and colon